src/Route/PathInfoBuilder.php

```diff
@@ -47,7 +47,7 @@ public function build(AdminInterface $admin, RouteCollection $collection)
         if ($this->manager->hasReader($admin->getClass())) {
             $collection->add('history', sprintf('%s/history', $admin->getRouterIdParameter()));
             $collection->add('history_view_revision', sprintf('%s/history/{revision}/view', $admin->getRouterIdParameter()));
-            $collection->add('history_compare_revisions', sprintf('%s/history/{base_revision}/{compare_revision}/compare', $admin->getRouterIdParameter()));
+            $collection->add('history_compare_revisions', sprintf('%s/history/{baseRevision}/{compareRevision}/compare', $admin->getRouterIdParameter()));
```

Comment:
Not sure if you should change the route in a stable release

Reply:
I'll try to check how could we deal with this.

Reply:
@core23, I added a BC layer for those cases where the param names are explicitly used.